steps/oneskyapp-pull-android-translations/0.0.2/step.yml

```diff
@@ -0,0 +1,75 @@
+title: Pull Android translations from an OneSkyApp project
+summary: Pull Android translations from an OneSkyApp project
+description: List all translations available on OneSkyApp and pull them into the current
+  project.
+website: https://github.com/prcaen/bitrise-step-oneskyapp-pull-translations
+source_code_url: https://github.com/prcaen/bitrise-step-oneskyapp-pull-translations
+support_url: https://github.com/prcaen/bitrise-step-oneskyapp-pull-translations/issues
+published_at: 2018-12-18T10:23:01.507167+01:00
+source:
+  git: https://github.com/prcaen/bitrise-step-oneskyapp-android-pull-translations.git
+  commit: 688ce65ac6349a739b4d9ffc197eddbcc9b9ae78
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
```

Comment:
Does this step work only for Android?
If so, you should specify the project_type_tags in the step.yml:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention